bases/node18.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "display": "Node 18",
+
+  "compilerOptions": {
+    "lib": ["es2022"],
```

Comment:
I don't think that TypeScript has an "es2022" lib?

ref: https://www.typescriptlang.org/tsconfig#lib
ref: https://github.com/microsoft/TypeScript/tree/main/lib

Does this work thru some other magic inside TypeScript?

Reply:
[Yes,](https://github.com/microsoft/TypeScript/blob/v4.7.2/src/compiler/commandLineParser.ts#L36) same goes for [target](https://github.com/microsoft/TypeScript/blob/v4.7.2/src/compiler/commandLineParser.ts#L332).

Reply:
Awesome, thanks!